embed/index.html

```diff
@@ -261,7 +261,38 @@ <h2>Listening to events</h2>
 got event {"type":"participant_join","detail":{"metadata":"userId"}}
 got event {"type":"participant_leave","detail":{"metadata":"userId"}}
 </output>
-<br> 
+
+<h2>Sending commands</h2>
+<blockquote class=warn>
+  <p>
+    For this feature to work, you must add the origin of your application to the "Allowed domains" section in your Whereby account. If not present, the following functions will not do anything.
+  </p>
+</blockquote>
+
+<p>The <code>&lt;whereby-embed&gt;</code> element exposes a set of functions your application can invoke in order to perform actions in the room. Currently, the following functions
+are available:
+
+<table>
+  <tr><th>Function <th>Description</tr>
```

Comment:
Maybe too nitpicky but what sounds better, "function" or "method"?

```suggestion
    For this feature to work, you must add the origin of your application to the "Allowed domains" section in your Whereby account. If not present, the following methods will not do anything.
  </p>
</blockquote>

<p>The <code>&lt;whereby-embed&gt;</code> element exposes a set of methods your application can invoke in order to perform actions in the room. Currently, the following methods
are available:

<table>
  <tr><th>Method <th>Description</tr>
```

Reply:
In a context of `class`es, methods are correct. But these are mostly used in langs such as Java, C# etc. For JS, I think functions are most widely used